### PR TITLE
fix: rehaul email landing page

### PR DESCRIPTION
**Pull request by jnikhila**
* Updated the content of the email landing page
* Added cards and icons for popular service providers
* Removed images that are no longer needed

## Checklist
I have:
- [x] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [x] added the meta description for each page in the PR
- [x] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [x] added images only when necessary
- [x] deleted the images that are no longer used for the updated pages in the PR
- [x] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [x] used the `<figure/>` tag instead of a markdown representation for images
- [x] added the `<figcaption/>` tag to add a caption to the image
- [x] added the `alt` attribute in the `` tag
- [x] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [x] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/email/README.md [hunk 1]
```diff
@@ -7,76 +7,88 @@ sidebar_position: 2
...
+* Invite users to join your Appsmith workspace by sending email invitations.
+* Reset password using password reset emails.
 
-:::note
-[Password resets](/help-and-support/troubleshooting-guide/application-errors#reset-password-error) for user accounts are handled by emails from the Appsmith instance. If you would like users to be able to perform this action, ensure that your instance has been configured to handle email notifications. 
-:::
+<VideoEmbed host="youtube" videoId="NOAofPbmJWw" title="Configure Email Service Provider." caption="Configure Email Service Provider" /> 
 
-Appsmith allows you to configure email using environment variables or the [admin settings](/getting-started/setup/instance-configuration).
+## Prerequisites
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-configuration/email/README.md:17]: this section doesn't give any information that is not already known. From the introduction, I think it's known that they need an Appsmith instance and an email service account. Do we need to mention these explicitly?  We don't mention this as prerequisites for any instance configuration guides.

File: website/docs/getting-started/setup/instance-configuration/email/README.md [hunk 2]
```diff
@@ -7,76 +7,88 @@ sidebar_position: 2
...
+   | `APPSMITH_MAIL_USERNAME` |	Set it to the username for accessing the SMTP service provider.
+   | `APPSMITH_MAIL_PASSWORD` |	Set it to the password for the SMTP user. Alternatively, you can use the API key generated by the email service provider for the SMTP user. |
 
-## Configuration guides
+3. Restart the instance.
 
-Follow the below guides to configure popular email service providers:
+## Troubleshooting
 
+If you have received a test email but can't send invitations then verify your email configuration again. If you continue to face issues, contact the support team using the chat widget at the bottom right of this page.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-configuration/email/README.md:88]: we can also add this link - https://docs.appsmith.com/help-and-support/troubleshooting-guide/deployment-errors#unable-to-send-emails

File: website/docs/getting-started/setup/instance-configuration/email/README.md [hunk 3]
```diff
@@ -7,76 +7,88 @@ sidebar_position: 2
...
 
-* [Sendgrid](sendgrid.md)
-* [Amazon-SES](amazon-ses.md)
-* [Gmail](gmail.md)
+## See also
```
Comment (danciaclara) [inline website/docs/getting-started/setup/instance-configuration/email/README.md:91]:
```suggestion
To configure email service for popular email service providers, see the guides below:
```

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/email/README.md (hunk 1 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md:17]: We do use it. We've used it in the:
* [External MongoDB](https://docs.appsmith.com/getting-started/setup/instance-configuration/custom-mongodb-redis#prerequisites).
* [Google Maps](https://docs.appsmith.com/getting-started/setup/instance-configuration/google-maps#prerequisites)

Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md:17]: They need to be able to access the instance to configure.
May be I can rewrite it as superuser.

File: website/docs/getting-started/setup/instance-configuration/email/README.md (hunk 3 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md:91]: We never add text to the see also or Further reading section, and it's self explanatory.

File: website/docs/getting-started/setup/instance-configuration/email/README.md (hunk 1 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md]: This is added to the steps so removed the prerequisite section now.

File: website/docs/getting-started/setup/instance-configuration/email/README.md (hunk 2 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md:82]: I searched for it, but didn't expect this to be under deployment errors :)

File: website/docs/getting-started/setup/instance-configuration/email/README.md (hunk 3 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/email/README.md:85]: Also, I feel Further reading as section heading is more appropriate here.

**Review by danciaclara — APPROVED** [review]

**Pull request merged**